src/pages/contact-us/SendGridForm/index.tsx

```diff
@@ -34,6 +34,8 @@ export default function (): React.ReactElement {
   const [email, setEmail] = useState<Field>({ val: "" });
   const [message, setMessage] = useState<Field>({ val: "" });
```

Comment:
Merge all of these form state into an big object, use `useReducer` instead of `useReducer` to manage state.